__tests__/components/forecast-summary.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ForecastSummary from '../../src/components/forecast-summary';
+
```

Comment:
You're repeating the same shallow wrapper code. Did you try putting it into a `forEach()`?